tox.ini

```diff
@@ -10,6 +10,6 @@ max-line-length = 99
 exclude = tests/__init__.py
 ignore = E722, W503
 
-[testenv:py27-flake8]
-deps = flake8
-commands = flake8 pyuntl setup.py tests
+[testenv:py37-flake8]
+deps=flake8
```

Comment:
This and the next line are inconsistent with previous lines. We should probably add the spaces back.